probinet/input/loader.py

```diff
@@ -11,7 +11,7 @@
 import networkx as nx
```

Comment:
Imports are not sorted properly, I believe.

Reply:
I will take care of this!